spec/models/reservations_validations_spec.rb

```diff
@@ -6,6 +6,8 @@
 
   let(:now) { Time.zone.now }
 
+  it { is_expected.to validate_presence_of(:reserve_start_at) }
```

Comment:
should we add in validation expectations for `product_id`, `order_detail_id` while we're in here?